{"uid":"b73a53a7bebd0ce2","name":"ValidationOfResults","historyId":"GalileoE1PcpsAmbiguousAcquisitionGSoCTest:GalileoE1PcpsAmbiguousAcquisitionGSoCTest#ValidationOfResults","time":{"start":1734688546120,"stop":1734688546124,"duration":4},"status":"failed","statusMessage":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","statusTrace":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":2,"retriesStatusChange":false,"beforeStages":[],"testStage":{"steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},"afterStages":[],"labels":[{"name":"resultFormat","value":"junit"},{"name":"suite","value":"GalileoE1PcpsAmbiguousAcquisitionGSoCTest"},{"name":"testClass","value":"GalileoE1PcpsAmbiguousAcquisitionGSoCTest"},{"name":"package","value":"GalileoE1PcpsAmbiguousAcquisitionGSoCTest"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"97a3d9f353d4857b","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734687850115,"stop":1734687850120,"duration":5}},{"uid":"d9eecb6ef46805d","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-test/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734449976367,"stop":1734449976372,"duration":5}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":26,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":26},"items":[{"uid":"41c9c68362f9e6a6","reportUrl":"https://xguerreropau.github.io/gnss-sdr/22//#testresult/41c9c68362f9e6a6","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"8837a7fbd841dd4","reportUrl":"https://xguerreropau.github.io/gnss-sdr/21//#testresult/8837a7fbd841dd4","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"bd1f524cefb51946","reportUrl":"https://xguerreropau.github.io/gnss-sdr/20//#testresult/bd1f524cefb51946","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"ce71021df46f7c4b","reportUrl":"https://xguerreropau.github.io/gnss-sdr/19//#testresult/ce71021df46f7c4b","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"fcc4ba78de93991f","reportUrl":"https://xguerreropau.github.io/gnss-sdr/18//#testresult/fcc4ba78de93991f","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"7aef6901484a0a62","reportUrl":"https://xguerreropau.github.io/gnss-sdr/17//#testresult/7aef6901484a0a62","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"a7cf6efe90f3f41d","reportUrl":"https://xguerreropau.github.io/gnss-sdr/16//#testresult/a7cf6efe90f3f41d","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"6c1bc96f2768bf8d","reportUrl":"https://xguerreropau.github.io/gnss-sdr/15//#testresult/6c1bc96f2768bf8d","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"522828948dccf8e9","reportUrl":"https://xguerreropau.github.io/gnss-sdr/14//#testresult/522828948dccf8e9","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"5353d9ee41917318","reportUrl":"https://xguerreropau.github.io/gnss-sdr/13//#testresult/5353d9ee41917318","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"4862a434ba79d070","reportUrl":"https://xguerreropau.github.io/gnss-sdr/12//#testresult/4862a434ba79d070","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"470d4d7da9af7e56","reportUrl":"https://xguerreropau.github.io/gnss-sdr/11//#testresult/470d4d7da9af7e56","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"a1965656d2590490","reportUrl":"https://xguerreropau.github.io/gnss-sdr/10//#testresult/a1965656d2590490","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"cd913d142d45b3e2","reportUrl":"https://xguerreropau.github.io/gnss-sdr/9//#testresult/cd913d142d45b3e2","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"1d1b3563dedd1aea","reportUrl":"https://xguerreropau.github.io/gnss-sdr/8//#testresult/1d1b3563dedd1aea","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"8a1a16a7c22114f5","reportUrl":"https://xguerreropau.github.io/gnss-sdr/7//#testresult/8a1a16a7c22114f5","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"e6af9fac1a1b0c41","reportUrl":"https://xguerreropau.github.io/gnss-sdr/6//#testresult/e6af9fac1a1b0c41","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"fbdadefc1510c602","reportUrl":"https://xguerreropau.github.io/gnss-sdr/3//#testresult/fbdadefc1510c602","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"a7e77b66d7554b89","reportUrl":"https://xguerreropau.github.io/gnss-sdr/2//#testresult/a7e77b66d7554b89","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}},{"uid":"456ab6b042bf1efe","reportUrl":"https://xguerreropau.github.io/gnss-sdr/1//#testresult/456ab6b042bf1efe","status":"failed","statusDetails":"/home/xavier/gnss-sdr/tests/unit-tests/signal-processing-blocks/acquisition/galileo_e1_pcps_ambiguous_acquisition_gsoc_test.cc:290\nExpected: { std::string path = std::string(\"/home/xavier/gnss-sdr/build-debug/thirdparty/\"); std::string file = path + \"signal_samples/Galileo_E1_ID_1_Fs_4Msps_8ms.dat\"; const char* file_name = file.c_str(); gr::blocks::file_source::sptr file_source = gr::blocks::file_source::make(sizeof(gr_complex), file_name, false); top_block->connect(file_source, 0, acquisition->get_left_block(), 0); top_block->msg_connect(acquisition->get_right_block(), pmt::mp(\"events\"), msg_rx, pmt::mp(\"events\")); } doesn't throw an exception.\n  Actual: it throws std::runtime_error with description \"can't open file\".\nFailure connecting the blocks of acquisition test.\n","time":{"start":1734688546120,"stop":1734688546124,"duration":4}}]},"tags":[]},"source":"b73a53a7bebd0ce2.json","parameterValues":[]}